### PR TITLE
Devise Style's

### PR DESCRIPTION
**Pull request by DevLuis123**
Add styles to all Devise's User model generated views

### CODE REVIEW

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+      <%= render "devise/shared/links" %>
+
+      <div>
+        <%= f.submit "Log in" , class: 'btn_log' %>
+      </div>
+
+    </div>
+  </div>
+<% end %>
```
Comment (MauricioNC) [inline app/views/devise/sessions/new.html.erb]: Add a new line at the end of your code

File: app/views/devise/unlocks/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+      <%= render "devise/shared/links" %>
+
+      <div>
+        <%= f.submit "Log in uwu" , class: 'btn-4'%>
+      </div>
+
+    </div>
+  </div>
+<% end %>
```
Comment (MauricioNC) [inline app/views/devise/unlocks/new.html.erb]: Same here.
Add a new line at the end of your code

**Review by MauricioNC — CHANGES_REQUESTED** [review]

**Review by juperameza — APPROVED** [review]
lgtm just attend @MauricioNC comments

File: app/views/devise/sessions/new.html.erb (hunk 1 above)
Comment (DevLuis123) [reply app/views/devise/sessions/new.html.erb]: fxd !!!

File: app/views/devise/unlocks/new.html.erb (hunk 2 above)
Comment (DevLuis123) [reply app/views/devise/unlocks/new.html.erb]: fxd !!

**Review by MauricioNC — APPROVED** [review]
LGTM

**Pull request merged**